frontend/components/search/ComboResults.vue

```diff
@@ -20,7 +20,10 @@
               :card-name="name"
             >
               <div class="card-name pl-3 pr-3">
-                {{ name }}
+                <strong v-if="isMissingInDecklist(name)" class="text-red-800"
```

Comment:
Could the class be conditional and we just style them appropriately? You could add the ` (not in deck)` in the CSS `:after` pseudo-element.